### ISSUE TITLE
Provide a CHANGELOG.md file and git tags

### ISSUE THREAD

**Issue by sonallux** [spotify/spotify-web-api-ts-sdk#54]
I enjoy using this library. Thanks for creating it 👍 

Unfortunately, it is currently hard to follow the changes made to the code for every release. Therefore I would suggest adding a `CHANGELOG.md` file, following the [keep a changelog format](https://keepachangelog.com/en/1.0.0/). And it would be great if git tags could be created and pushed when a release is published. So everyone can look up the exact repository content used to publish a release.

**Comment by Jordan-Loeser** [comment]
Fixed in #100!
